MediaPlayer/MediaSliderDecorator.js

```diff
@@ -105,13 +105,12 @@ const MediaSliderDecorator = hoc((config, Wrapped) => {
 
 		componentDidUpdate (prevProps, prevState) {
 			if (prevState.x !== this.state.x) {
-				forward('onKnobMove', this.getEventPayload('onKnobMove'), this.props);
+				forwardCustom('onKnobMove')(this.getEventPayload(), this.props);
 			}
 		}
 
-		getEventPayload (type) {
+		getEventPayload () {
```

Comment:
I'm seeing one more place to use this function. Does this change affect line 62?

Reply:
Nice catch!
Change of line 112 affects on line 62 so that type value of `forward('onChange')` is changed from `'keyup'` to `undefined`, and both values are wrong! I will update `forward` to `forwardCustom` to handle its type value in event payload correctly.

Reply:
✅